docs/ABOUT.md

```diff
@@ -1,8 +1,16 @@
 # About
 
-Lisp Flavoured Erlang (AKA LFE) is a lisp syntax front-end to the Erlang compiler. 
+**Lisp Flavoured Erlang** (AKA LFE) is a lisp syntax **front-end** to the **Erlang compiler**. 
 Code produced with it is compatible with "normal" Erlang code. 
 LFE is a (proper) Lisp based on the features and limitations of the Erlang VM.
```

Comment:
This sentence is a bit vague. What is proper? What are the features and limitations of the Erlang VM? This doesn't add any information to the last two sentences so perhaps it's best to remove it.